.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - run: yarn nx affected --target=type-check
       - run: yarn nx affected --target=lint
       - run: yarn nx affected --target=test
-      - run: yarn nx affected --target=component-test --skipInstall
+      - run: yarn nx affected --target=component-test --skipInstall --parallel --max-parallel=1
```

Comment:
@hotell

Looks like playwright is not close the server after each target tested, when running with `nx affected`.

Playwright first reports: `Dev Server is already running at http://localhost:3100/, using it`, later fails with several errors: "Error: Could not connect to the server", "Error: Unregistered component". Happens locally as well with `nx affected`.